charts/argoconfig/Chart.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: v2
 name: argoconfig
-description: Configure Argo CD AppProjects, Applications and more
+description: Configure Argo CD AppProjects and Applications
 type: library
-version: 0.6.3
-appVersion: latest
```

Comment:
isn't `appVersion` needed anymore?

Reply:
library charts do not have an appVersion by definition.